packs/st2/actions/workflows/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,37 @@
+---
+  chain:
+    -
+      name: "check_for_permission_to_send_anonymous_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::collect_anonymous_data"
+      on-success: "get_server_uuid"
+    -
+      name: "get_server_uuid"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::server_uuid"
+      publish:
+        server_uuid: "{{get_server_uuid.result}}"
+      on-success: "generate_anonymous_data"
+    -
+      name: "generate_anonymous_data"
+      ref: "core.local"
+      params:
+        cmd: "st2-submit-debug-info --exclude-configs --exclude-logs --exclude-content --review"
+      publish:
+        tarball_logline: "{{generate_anonymous_data.stdout}}"
+      on-success: "get_tarball_location"
+    -
+      name: "get_tarball_location"
+      ref: "core.local"
+      params:
+        cmd: "echo '{{tarball_logline}}' | grep tmp | awk '{print $(NF)}'"
+      publish:
+        tarball: "{{get_tarball_location.stdout}}"
+      on-success: "upload_to_s3"
+    -
+      name: "upload_to_s3"
+      ref: "core.local_sudo"
+      params:
+        cmd: "/opt/stackstorm/virtualenvs/st2/bin/s3 -c /opt/stackstorm/packs/st2/config.yaml -b st2express-install put {{tarball}}"
```

Comment:
The base_path '/opt/stackstorm' is confgurable so this will mostly likely to lead to a bug at some point.

Perhaps you should consider writing a python action in the st2 pack that does not have to make such assumptions since it would work in the `st2` virtualenv.